dataframely/testing/const.py

```diff
@@ -47,3 +47,7 @@
 ALL_COLUMN_TYPES: list[type[dc.Column]] = (
     [dc.Any] + COLUMN_TYPES + SUPERTYPE_COLUMN_TYPES
 )
+
+NO_VALIDATION_COLUMN_TYPES: list[type[dc.Column]] = [
```

Comment:
Is this supposed to be all column types which do not have a validation rule by default or is there another idea? Could you add a comment what this is and why it exists?